docs/architecture/asciidoc/en/_description_modules.ad

```diff
@@ -0,0 +1,26 @@
+== Description of the different modules
+
+Here are the different modules of this project and their role (see the sections below for more details)
+
+* **gateway :** The gateway is our rest server which provides our rest API.
+* **grafana-historian-datasource :** It is a git sub-module which is our Grafana plugin to expose a dedicated datasource . We will use it for viewing on historian timeseries. The datasource serves as a junction between Grafana and the gateway (REST API).
```

Comment:
This module does not exist anymore. We develop the plugin only in his specific project
```suggestion
```